packages/renderer/src/lib/node/NodeEmptyScreen.svelte

```diff
@@ -17,6 +18,15 @@ function getText(state: ContextGeneralState | undefined): string {
 }
 
 $: text = getText($kubernetesCurrentContextState);
+$: currentContextName = $kubernetesContexts.find(c => c.currentContext)?.name;
+
+function checkContext(): void {
+  if (currentContextName) {
+    window.kubernetesRefreshContextState(currentContextName);
+  }
+}
 </script>
 
-<EmptyScreen icon={NodeIcon} title="No nodes" message={text} />
+<EmptyScreen icon={NodeIcon} title="No nodes" message={text}>
+  <button onclick={checkContext}>Check connection now</button>
```

Comment:
I think we don't need the now as the button implies that it would be when we click on it

method being refresh then maybe just `Refresh`

Reply:
This is really to test the backend part, I'm not sure at all if it will be the implementation we want to do in the frontend